src/common/commandExecutor.ts

```diff
@@ -10,13 +10,20 @@ import {HostPlatform, HostPlatformId} from "../common/hostPlatform";
 import {ErrorHelper} from "./error/errorHelper";
 import {InternalErrorCode} from "./error/internalErrorCode";
 
+export enum CommandVerbosity {
+    OUTPUT,
+    SILENT,
+    PROGRESS,
+}
+
 interface EnvironmentOptions {
     REACT_DEBUGGER?: string;
 }
 
 interface Options {
     env?: EnvironmentOptions;
-    silent?: boolean;
```

Comment:
This is replaced with SILENT in CommandVerbosity, but I do not see it's used by the logger. Will keep looking at other places. Possibly in Log class. For now, leave a note here.